app/controllers/placements/support/support_users_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Placements::Support::SupportUsersController < Placements::Support::ApplicationController
+  before_action :set_support_user, only: %i[show]
+
+  def index
+    @support_users = Placements::SupportUser.order(created_at: :desc)
```

Comment:
I assume we won't have too many support users to require pagination?

Reply:
That would be my assumption. This is extracted some the same implementation on Claim, so I'm guessing that this is ok. There also doesn't appear to be pagination on the prototype

Reply:
There is pagination on the support#org#index page, but you need to create over 25 organisations to get it to show. It's OK, if we find a need we can add it later 🚀